cmd/admin-heal.go

```diff
@@ -66,6 +77,11 @@ USAGE:
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
+
+SCAN MODES:
+   normal: Search for missing and inconsistent data in the disks
+     deep: Cover normal scan and additionally search for data corruption
```

Comment:
This would be better?:
```
 normal (default): Heal objects which are missing on one or more disks
 deep            : Heal objects which are missing on one or more disks and heal objects with bitrot
```